pipeline_2d_1d.jl

```diff
@@ -466,7 +458,7 @@ all1Da = replace.(all2Dperchip[1], "ar2D" => "ar1D")
 println("Reinterpolating exposure spectra:");
 flush(stdout);
 @everywhere reinterp_spectra_partial(fname) = reinterp_spectra(
-    fname, backupWaveSoln = night_wave_soln)
+    fname, roughwave_dict,backupWaveSoln = night_wave_soln)
```

Comment:
Why did this change?

Reply:
uh, because reinterp_spectra was leveraging a global that was no longer in scope when I refactored.